target/linux/ramips/dts/mt7621.dtsi

```diff
@@ -124,9 +124,6 @@
 			compatible = "ralink,mt7621-systick", "ralink,cevt-systick";
```

Comment:
You can remove the whole node if you want. There is no sense in having this for mt7621 based boards.

Reply:
Thank you, I have removed it.